<html><head>     <meta http-equiv="Content-Type" content="text/html; charset=ISO-8859-1">
     <meta name="Generator" content="Eliza@yasara.com">     <title>Experiment - Choose and control experiments</title>
     <link rel="stylesheet" href="Experiment%20-%20Choose%20and%20control%20experiments_files/stylesh.css" type="text/css">   </head>   <body bgproperties="fixed" topmargin="0" leftmargin="0" background="bgcmbi.gif" bgcolor="#FFFFFF" marginheight="0" marginwidth="0">   <script language="JavaScript"> 
<!--
 if (document.images)
{ ButtonTop=new Image(190,36);ButtonTop.src="ButtonTop.gif";   ButtonTopAct=new Image(190,
36);ButtonTopAct.src="ButtonTopAct.gif";   ButtonUp=new Image(190,36);ButtonUp.src=
"ButtonUp.gif";   ButtonUpAct=new Image(190,36);ButtonUpAct.src="ButtonUpAct.gif";
  ButtonPrev=new Image(190,36);ButtonPrev.src="ButtonPrev.gif";   ButtonPrevAct=new Image(190,
36);ButtonPrevAct.src="ButtonPrevAct.gif";   ButtonNext=new Image(190,36);ButtonNext.src=
"ButtonNext.gif";   ButtonNextAct=new Image(190,36);ButtonNextAct.src="ButtonNextAct.gif";
  ButtonUnfold=new Image(190,36);ButtonUnfold.src="ButtonUnfold.gif";   ButtonUnfoldAct=new Image(190,
36);ButtonUnfoldAct.src="ButtonUnfoldAct.gif"; } function swapimg(imgname,objsrc)
{ if (document.images)   { document.images[imgname].src=eval (objsrc + ".src");
} } 
// -->
   </script>   <table border="0" cellpadding="0" cellspacing="0" width="100%"><tbody><tr> <td bgcolor="#7676EC" height="60" width="200"><img src="Experiment%20-%20Choose%20and%20control%20experiments_files/border1.gif" height="60" width="200"></td> <td valign="bottom" width="100%"><div align="left"><table border="0" cellpadding="0" cellspacing="0"><tbody><tr> <td bgcolor="#7676EC" valign="bottom" width="96"><img src="Experiment%20-%20Choose%20and%20control%20experiments_files/spacer96.gif" height="1" width="96"></td> <td bgcolor="#7676EC" valign="bottom"><img src="Experiment%20-%20Choose%20and%20control%20experiments_files/border6.gif" alt="Endonuclease PvuII (1PVI) " height="50" width="324"></td> <td bgcolor="#7676EC" valign="bottom" width="110"><img src="Experiment%20-%20Choose%20and%20control%20experiments_files/spacer11.gif" height="1" width="110"></td> <td><img src="Experiment%20-%20Choose%20and%20control%20experiments_files/border8.gif" alt="DNA - GATTACAGATTACA" height="60" width="116"></td></tr></tbody></table> </div></td></tr><tr> <td align="right" bgcolor="#7676EC" height="104"><img src="Experiment%20-%20Choose%20and%20control%20experiments_files/border2.gif" alt="CAP - Catabolite gene Activating Protein (1BER)" height="104" width="150"></td> <td><table border="0" cellpadding="0" cellspacing="0"><tbody><tr>
 <td><img src="Experiment%20-%20Choose%20and%20control%20experiments_files/border4.gif" alt="DNA - GATTACAGATTACAGATTACA" height="104" width="96"></td> <td><img src="Experiment%20-%20Choose%20and%20control%20experiments_files/border10.gif" alt="Endonuclease PvuII bound to palindromic DNA recognition site CAGCTG (1PVI) " height="104" width="324"></td> <td><img src="Experiment%20-%20Choose%20and%20control%20experiments_files/border7.gif" alt="DNA - GATTACAGATTACAGATTACA" height="104" width="110"></td> <td><img src="Experiment%20-%20Choose%20and%20control%20experiments_files/border9.gif" alt="TBP - TATA box Binding Protein (1C9B)" height="104" width="116"></td></tr></tbody></table></td></tr><tr> <td height="36"><img src="Experiment%20-%20Choose%20and%20control%20experiments_files/border11.gif" alt="CAP - Catabolite gene Activating Protein (1BER)" height="36" width="200"></td>
 <td valign="top" width="100%"><img src="Experiment%20-%20Choose%20and%20control%20experiments_files/border5.gif" height="32" width="32"></td>
</tr><tr> <td height="60" valign="top"><table border="0" cellpadding="0" cellspacing="0" width="100%"><tbody><tr> <td><img src="Experiment%20-%20Choose%20and%20control%20experiments_files/border3.gif" alt="GCN4 - leucine zipper transcription factor bound to palindromic DNA recognition site ATGAC(G)TCAT (1YSA)" height="60" width="200"></td></tr><tr> <td> <img src="Experiment%20-%20Choose%20and%20control%20experiments_files/ButtonTop.gif" name="ButtonTop" border="0" height="36" width="190"></td></tr><tr> <td><img src="Experiment%20-%20Choose%20and%20control%20experiments_files/border12.gif" alt="GCN4 - leucine zipper transcription factor bound to palindromic DNA recognition site ATGAC(G)TCAT (1YSA)" height="78" width="190"></td></tr><tr> <td> <img src="Experiment%20-%20Choose%20and%20control%20experiments_files/ButtonUp.gif" name="ButtonUp" border="0" height="36" width="190"></td></tr><tr> <td><img src="Experiment%20-%20Choose%20and%20control%20experiments_files/border12.gif" alt="GCN4 - leucine zipper transcription factor bound to palindromic DNA recognition site ATGAC(G)TCAT (1YSA)" height="78" width="190"></td></tr><tr> <td> <img src="Experiment%20-%20Choose%20and%20control%20experiments_files/ButtonPrev.gif" name="ButtonPrev" border="0" height="36" width="190"></td></tr> <tr> <td><img src="Experiment%20-%20Choose%20and%20control%20experiments_files/border12.gif" alt="GCN4 - leucine zipper transcription factor bound to palindromic DNA recognition site ATGAC(G)TCAT (1YSA)" height="78" width="190"></td></tr><tr> <td> <img src="Experiment%20-%20Choose%20and%20control%20experiments_files/ButtonNext.gif" name="ButtonNext" border="0" height="36" width="190"></td></tr> <tr> <td><img src="Experiment%20-%20Choose%20and%20control%20experiments_files/border12.gif" alt="GCN4 - leucine zipper transcription factor bound to palindromic DNA recognition site ATGAC(G)TCAT (1YSA)" height="78" width="190"></td></tr><tr> <td> <img src="Experiment%20-%20Choose%20and%20control%20experiments_files/ButtonUnfold.gif" name="ButtonUnfold" border="0" height="36" width="190"></td></tr>  <tr> <td><img src="Experiment%20-%20Choose%20and%20control%20experiments_files/border13.gif" alt="TBP - TATA box Binding Protein (1C9B)" height="155" width="190">
</td></tr>  <tr> <td>&nbsp;</td></tr></tbody></table> </td>  <td colspan="5" valign="top" width="100%">  <table><tbody><tr><td valign="top" width="0%"><h4 align="left"><font color="#000000">°&nbsp;</font>
<font color="#ffffff"></font></h4></td><td valign="top" width="0%%"><h4>Experiment</h4></td><td valign="top" width="0%"><h4 align="center">
<font color="#000000">-</font></h4></td><td valign="top" width="100%%"><h4 align="left">Choose and control experiments</h4></td></tr></tbody></table> <br>
<table border="0" cellpadding="2" cellspacing="0"><tbody><tr><td colspan="15" bgcolor="#666600"><img src="Experiment%20-%20Choose%20and%20control%20experiments_files/spacer.gif" height="8"></td></tr><tr><td bgcolor="#675e13"><img src="Experiment%20-%20Choose%20and%20control%20experiments_files/spacer.gif" width="8"></td><td bgcolor="#675e13"><img src="Experiment%20-%20Choose%20and%20control%20experiments_files/spacer.gif" width="4"></td><td bgcolor="#675e13">
<img src="Experiment%20-%20Choose%20and%20control%20experiments_files/spacer.gif" width="4"></td><td>Command</td><td bgcolor="#675e13"><img src="Experiment%20-%20Choose%20and%20control%20experiments_files/spacer.gif" width="4"></td><td>Argument</td><td bgcolor="#675e13"><img src="Experiment%20-%20Choose%20and%20control%20experiments_files/spacer.gif" width="4"></td><td>Datatype</td><td bgcolor="#675e13"><img src="Experiment%20-%20Choose%20and%20control%20experiments_files/spacer.gif" width="4"></td><td>Default</td>
<td bgcolor="#675e13"><img src="Experiment%20-%20Choose%20and%20control%20experiments_files/spacer.gif" width="4"></td><td>Min</td><td bgcolor="#675e13">
<img src="Experiment%20-%20Choose%20and%20control%20experiments_files/spacer.gif" width="4"></td><td>Max</td><td bgcolor="#675e13"><img src="Experiment%20-%20Choose%20and%20control%20experiments_files/spacer.gif" width="8"></td></tr><tr><td colspan="15" bgcolor="#695627"><img src="Experiment%20-%20Choose%20and%20control%20experiments_files/spacer.gif" height="4"></td>
</tr><tr><td bgcolor="#6b4e3a"><img src="Experiment%20-%20Choose%20and%20control%20experiments_files/spacer.gif" width="8"></td><td valign="top">Format:</td>
<td bgcolor="#6b4e3a"><img src="Experiment%20-%20Choose%20and%20control%20experiments_files/spacer.gif" width="4"></td><td valign="top">Experiment</td>
<td></td><td>Minimization | Neutralization | Morphing | Docking | NMRFolding | On
| Pause | Continue | Off</td><td bgcolor="#6b4e3a"><img src="Experiment%20-%20Choose%20and%20control%20experiments_files/spacer.gif" width="4"></td>
<td>STRING</td><td bgcolor="#6b4e3a"><img src="Experiment%20-%20Choose%20and%20control%20experiments_files/spacer.gif" width="4"></td><td>-</td>
<td bgcolor="#6b4e3a"><img src="Experiment%20-%20Choose%20and%20control%20experiments_files/spacer.gif" width="4"></td><td>-</td><td bgcolor="#6b4e3a">
<img src="Experiment%20-%20Choose%20and%20control%20experiments_files/spacer.gif" width="4"></td><td>-</td><td bgcolor="#6b4e3a"><img src="Experiment%20-%20Choose%20and%20control%20experiments_files/spacer.gif" width="8"></td></tr><tr><td colspan="15" bgcolor="#6d464e"><img src="Experiment%20-%20Choose%20and%20control%20experiments_files/spacer.gif" height="8"></td>
</tr><tr><td bgcolor="#6f3e62"><img src="Experiment%20-%20Choose%20and%20control%20experiments_files/spacer.gif" width="8"></td><td>Menu:</td><td bgcolor="#6f3e62"><img src="Experiment%20-%20Choose%20and%20control%20experiments_files/spacer.gif" width="4"></td><td colspan="11">Options &gt; Control experiment</td>
<td bgcolor="#6f3e62"><img src="Experiment%20-%20Choose%20and%20control%20experiments_files/spacer.gif" width="8"></td></tr><tr><td colspan="15" bgcolor="#713675"><img src="Experiment%20-%20Choose%20and%20control%20experiments_files/spacer.gif" height="8"></td></tr><tr><td bgcolor="#732f89"><img src="Experiment%20-%20Choose%20and%20control%20experiments_files/spacer.gif" width="8"></td><td>Python:</td><td bgcolor="#732f89"><img src="Experiment%20-%20Choose%20and%20control%20experiments_files/spacer.gif" width="4"></td><td colspan="11">Experiment(noname1)<br>ExperimentMinimization(convergence=None)<br>ExperimentNeutralization(waterdensity=None,
 nacl=None, ph=None, pkafile=None, speed=None)<br>ExperimentMorphing(startobj=None,
 endobj=None, structures=None, structurefile=None, morphforce=None)<br>ExperimentDocking(method=None,
 ligandobj=None, receptorobj=None, runs=None, clusterrmsd=None, resultfile=None,
 gridparlist=None, dockparlist=None)<br>ExperimentHomologyModeling(sequencefile=None,
 psiblasts=None, evalue=None, oligostate=None, templates=None, alignments=None, alignfile=None,
 templateobj=None, loopsamples=None, speed=None, animation=None, resultfile=None,
 termextension=None, residues=None, structprofile=None, fixmodelres=None, looplenmax=None)<br>ExperimentNMRFolding(startobj=None,
 restrainfile=None, structures=None, structurefile=None)</td><td bgcolor="#732f89">
<img src="Experiment%20-%20Choose%20and%20control%20experiments_files/spacer.gif" width="8"></td></tr><tr><td colspan="15" bgcolor="#75279c"><img src="Experiment%20-%20Choose%20and%20control%20experiments_files/spacer.gif" height="8"></td></tr><tr><td bgcolor="#771fb0"><img src="Experiment%20-%20Choose%20and%20control%20experiments_files/spacer.gif" width="8"></td><td>Keys:</td><td bgcolor="#771fb0"><img src="Experiment%20-%20Choose%20and%20control%20experiments_files/spacer.gif" width="4"></td><td colspan="11">&lt;Pause&gt; or &lt;P&gt;&nbsp;&nbsp;-&nbsp;&nbsp;Pause/continue all tasks, including the experiment<br>
</td><td bgcolor="#771fb0"><img src="Experiment%20-%20Choose%20and%20control%20experiments_files/spacer.gif" width="8"></td></tr><tr><td colspan="15" bgcolor="#7917c4"><img src="Experiment%20-%20Choose%20and%20control%20experiments_files/spacer.gif" height="8"></td></tr><tr><td bgcolor="#7b0fd7"><img src="Experiment%20-%20Choose%20and%20control%20experiments_files/spacer.gif" width="8"></td><td>Required:</td><td bgcolor="#7b0fd7"><img src="Experiment%20-%20Choose%20and%20control%20experiments_files/spacer.gif" width="4"></td><td colspan="11"><img src="Experiment%20-%20Choose%20and%20control%20experiments_files/yadyna18.gif" align="left" border="0" height="32" width="184">
</td><td bgcolor="#7b0fd7"><img src="Experiment%20-%20Choose%20and%20control%20experiments_files/spacer.gif" width="8"></td></tr><tr><td colspan="15" bgcolor="#7d07eb"><img src="Experiment%20-%20Choose%20and%20control%20experiments_files/spacer.gif" height="8"></td></tr></tbody></table><br><br><div class="jr13">Experiments
 are YASARA's way of handling complicated things at the touch of a 
button. During an experiment,
 YASARA takes over control. As in real life, you should not interfere 
too much while YASARA is cooking. E.g. if you try to delete some atoms 
or change the simulation parameters in the middle of a
'Neutralization Experiment', YASARA will ask you to stop the experiment 
first. Like molecular dynamics simulations,
 experiments cannot be undone.</div><p></p><div class="jr13">When you 
start an experiment
('Experiment On'), YASARA performs various checks and reassigns the 
force field parameters. If you want to modify the force field,
 e.g. by adding distance constraints or modifying atom
charges, do that immediately after starting the experiment,
 not before. Otherwise the changes may be undone.</div><p></p><div class="jr13">The
'Experiment
 Pause' command only pauses YASARA's control over the scene, e.g. if a 
simulation is running as part of the experiment,
 it will continue running until it is paused separately
 or until YASARA is paused completely by pressing the &lt;Pause&gt; key.</div><br>
<h5><font color="#0000ff">Example 1:</font></h5>
<pre>Experiment Minimization</pre>
<br><div class="jr13">Prepare an energy minimization experiment, see example macro below. YASARA creates a
cell
 and starts with a steepest descent minimization using the current 
simulation parameters. Regions with strong bumps are initially minimized
 without electrostatic interactions to avoid
short-range energy traps. As soon as the bumps have been removed and the worst conformational stress is gone
(i.e. when the square root of MinStep/AccelMax
 becomes greater than
2.5 fs, with AccelMax being the maximum acceleration acting on any atom 
in the soup),
 a simulated annealing minimization is done until the energy (calculated
 every 200 steps) converges. The convergence criterion can be set with 
the
'Convergence' parameter. This procedure thus moves the soup to a deep 
energy minimum in the neighborhood,
 which is not necessarily the global energy minimum. To describe this 
method in a materials
&amp; methods section, the following text could be used: "To remove 
bumps and correct the covalent geometry,
 the structure was energy-minimized with the .... force field [ref
], using a ... A force cutoff and (if applied) the Particle Mesh Ewald algorithm
[ref]
 to treat longrange electrostatic interactions. After removal of 
conformational stress by a short steepest descent minimization,
 the procedure continued by simulated annealing (time step 2 fs, atom 
velocities scaled down by
0.9 every 10th step) until convergence was reached, i.e. the energy 
improved by less than
0.05 kJ/mol per atom during 200 steps.".</div><hr><br><h5><font color="#0000ff">Example
2:</font></h5>
<pre>Experiment Neutralization</pre>
<br><div class="jr13">Prepare a cell neutralization and pKa prediction experiment,
 see example macro below. It is helpful to optimize the hydrogen bonding network
 before. YASARA then predicts pKa values for Asp, Glu, His and Lys residues using a
method that has been calibrated on experimental pKa measurements
 and achieves a high overall accuracy, but does usually not capture very large shifts of special active site residues
(resort to a different method for these and use the pKaRes
 command to override YASARA's predictions, also look at the pKa 
prediction result file for scientific details). These calculations are 
only possible with
periodic boundaries
 and consider all atoms in the simulation cell except waters. Based on 
the chosen pH,
 YASARA then assigns the protonation states. Asp and Glu are protonated 
if the predicted pKa is higher than the pH. His is protonated if the 
predicted pKa is higher than the pH and it does not accept a hydrogen 
bond,
 and deprotonated otherwise. Cys is deprotonated if the selected pH is 
higher than
8.7. Lys is deprotonated if the predicted pKa is lower than the pH. Tyr 
and Arg are not modified,
 because the (Y)AMBER force fields do not have default parameters for 
the deprotonated states. The pKa predictions are saved in a text file 
with
*.pka extension and can additionally be obtained with the pKaRes
 command. Then the simulation box is filled with water molecules, and 
Na/Cl counter ions are placed at the locations of the lowest/highest 
electrostatic potential until the cell is neutralized and the requested 
NaCl concentration is reached
(normally 0.9%). The location of the counter ions does not matter in 
practice, since they randomly diffuse away later. If you want to keep 
YASARA from touching some structurally important waters,
 fix at least the water oxygen
. Finally a short MD simulation is run for the solvent, and water 
molecules are subsequently deleted until the water density has reached 
the requested value. Note that the pH setting affects only the initial 
protonation states but has no additional influence during a simulation. 
When running simulations of the same protein at different pH values,
 use the Compare command to make sure that there are indeed different protonation states,
 otherwise you would simply run the same simulation multiple times.</div><hr><br>
<h5><font color="#0000ff">Example 3:</font></h5>
<pre>Experiment Morphing</pre>
<br><div class="jr13">Prepare a morphing experiment, see example macro 
below. YASARA transforms one structure into another during a simulated 
annealing minimization. This is done by applying forces that pull the 
atoms in the initial
(start) structure to their respective coordinates in the target (end) 
structure. The RMSD to the target is monitored,
 and the pulling forces are increased when the morphing process gets 
stuck in a local minimum. Snapshots can be saved to create a movie of 
the transformation. Morphing essentially requires that initial and 
target structures contain the same atoms in the same order
(i.e. same atom names, residue names and molecule names), nevertheless 
the target structure may contain fewer atoms. In this case no pulling 
force will be applied to those atoms that exist only in the initial 
structure. If you get an error message about atoms missing in the 
initial structure,
 use the CompareAtom
 command to get a complete list of all atoms that differ. While morphing
 between snapshots from MD simulations is trivial
(since these contain exactly the same atoms), morphing between two 
different PDB files usually requires manual intervention
(like renumbering residues or deleting atoms in the target structure 
that are missing in the initial structure). Most morphing difficulties 
arise when the initial and target structures are very different
(so that atoms must effectively pass through each other to reach the 
target), or when the target has incorrect geometry
(so that e.g. bonds must be stretched on the way to the target). The 
'MorphForce' parameter
(see example below) provides the means to fine-tune YASARA's behavior in
 these cases. Note that pulling too strong can lead to frequent 
simulation failures. To describe this method in a materials
&amp; methods section, the following text could be used: "A low-energy 
pathway between the two conformations was calculated with YASARA[ref] using the .... force field[ref
] with a ... A force cutoff. After removing clashes with an initial 
unrestrained steepest descent minimization,
 the procedure continued with a restrained simulated annealing 
minimization (time step
2 fs, atom velocities scaled down by 0.9 every 10th step). A force of 
100 Piconewton
(pointing towards the target coordinates) was applied to all the atoms 
and the RMSD from the target coordinates was measured every
10th step. If the RMSD improved less than 0.01 A, the pulling force was 
increased by
50%, till a maximum of 10000 pN.".</div><hr><br><h5><font color="#0000ff">Example
4:</font></h5>
<pre>Experiment Docking</pre>
<br><div class="jr13">Prepare a docking experiment in YASARA Structure. While YASARA's own docking module is still under development,
 this experiment invokes a fully integrated and customized version of Autodock 4. All the details
are described here and in the example macro below.</div>
<hr><br><h5><font color="#0000ff">Example 5:</font></h5>
<pre>Experiment HomologyModeling</pre>
<br><div class="jr13">Prepare a homology modeling experiment in YASARA Structure. All the details
are described here and in the example macro below.</div>
<hr><br><h5><font color="#0000ff">Example 6:</font></h5>
<pre>Experiment NMRFolding</pre>
<br><div class="jr13">Prepare an NMR folding experiment in YASARA Structure with NMR module,
 see example macro below. Note that YASARA Structure and the NMR Structure Determination Module are required. YASARA folds a
usually linear peptide chain
 to a globular conformation using distance- and dihedral angle 
restraints derived from an NMR spectrum. This is done quickly using 
internal coordinates,
 i.e. by changing torsion angles. The torsion angles are picked randomly
 from knowledge based potentials defined by one of the
YASARA force fields.
 During the folding process,
 the restraints are added sequentially, from short- to long-ranged ones.
 The result is not accurate,
 but captures the overall protein topology and can quickly be converted 
to a realistic structure by molecular dynamics refinement.</div>
<hr><br><h5><font color="#0000ff">Example 7:</font></h5>
<pre>Experiment On</pre>
<br><div class="jr13">Start the experiment chosen before.</div><hr><br><h5><font color="#0000ff">Example 8:</font></h5>
<pre>Experiment Pause</pre>
<br><div class="jr13">Temporarily hold a running experiment.</div><hr><br><h5><font color="#0000ff">Example 9:</font></h5>
<pre>Experiment Continue</pre>
<br><div class="jr13">Continue a paused experiment.</div><hr><br><h5><font color="#0000ff">Example 10:</font></h5>
<pre>Experiment Off</pre>
<br><div class="jr13">Stop the current experiment.</div><hr><br><br><h5><font color="#0000ff">Example macro 1:</font></h5>
<pre><font color="#000000"><font color="#ff0000"># EXAMPLE Experiment Morphing</font>
<font color="#ff0000"># Requires YASARA Dynamics</font>
Clear
<font color="#ff0000"># Load the starting structure</font>
start = LoadPDB 1crn
<font color="#ff0000"># Load the target structure and clean both</font>
<font color="#ff0000"># (Provide your own target structure here)</font>
end = LoadPDB 1crn_md
Clean
<font color="#ff0000"># Create cell and choose force field</font>
Cell Auto
ForceField AMBER03,SetPar=Yes
<font color="#ff0000"># Prepare morphing experiment</font>
Experiment Morphing
<font color="#ff0000">  # Initial start structure</font>
  StartObj (start)
<font color="#ff0000">  # Final end structure</font>
  EndObj (end)
<font color="#ff0000">  # Number of intermediate structures to save</font>
  Structures 10
<font color="#ff0000">  # Filename for intermediate structures (will be incremented)</font>
  StructureFile morph0000
<font color="#ff0000">  # Morphing force in PicoNewton</font>
<font color="#ff0000">  #   Min = minimum force (can be increased to speed up morphing)</font>
<font color="#ff0000">  #   Switch = force beyond which non-bonded interactions will be gradually switched off</font>
<font color="#ff0000">  #   Max = maximum force (morphing stops if it gets stuck and the maximum force has been reached)</font>
  MorphForce Min=100,Switch=2000,Max=10000
<font color="#ff0000"># Start experiment</font>
Experiment On
<font color="#ff0000"># Wait till end of experiment</font>
Wait ExpEnd
</font></pre>
  <table align="center"><tbody><tr><td width="400"><font color="#0000ff"><img src="Experiment%20-%20Choose%20and%20control%20experiments_files/ExperimentMorphing.jpg" align="middle" border="8"></font></td></tr><tr><td width="400">Figure: Result of the example macro
1 above.</td></tr></tbody></table> <br><hr><br><h5><font color="#0000ff">Example macro 2:</font>
</h5>
<pre><font color="#000000"><font color="#ff0000"># EXAMPLE Experiment Neutralization</font>
<font color="#ff0000"># Requires YASARA Dynamics</font>
<font color="#ff0000"># IMPORTANT: Look at the *.pka result file for further details and hints.</font>
Clear
LoadPDB 1crn
<font color="#ff0000"># Mutate one arginine to glutamate, net charge is now -2</font>
SwapRes Arg 17,Glu
Clean
Cell Auto
ForceField AMBER96
Cutoff 7.86
Boundary periodic
<font color="#ff0000"># Prepare neutralization experiment</font>
Experiment Neutralization
<font color="#ff0000">  # Fill the cell with water at a density of 1.0 g/cm^3</font>
  WaterDensity 1.0
<font color="#ff0000">  # And NaCl counter ions with 0.9%</font>
  NaCl 0.9
<font color="#ff0000">  # Protonate ionizable groups according to pH 7.0</font>
  pH 7.0
<font color="#ff0000">  # Save pKa predictions as dat/1crn_mutant.pka</font>
  pKaFile 1crn_mutant
<font color="#ff0000">  # Finish quickly (final water density will be OK, but not exact)</font>
  Speed Fast
<font color="#ff0000">  # If needed, a residue selection of vacuum in cell coordinates, where no water will be placed.</font>
<font color="#ff0000">  # The following example keeps a 30 A membrane layer consisting of POP residues empty:</font>
<font color="#ff0000">  # Vacuum LocalY&gt;15 LocalY&lt;45 and with distance&lt;5 from POP</font>
<font color="#ff0000"># Start experiment</font>
Experiment On
<font color="#ff0000"># Wait till end of experiment</font>
Wait ExpEnd
</font></pre>
  <table align="center"><tbody><tr><td width="400"><font color="#0000ff"><img src="Experiment%20-%20Choose%20and%20control%20experiments_files/ExperimentNeutralization.jpg" align="middle" border="8"></font></td></tr><tr><td width="400">Figure: Result of the example macro
2 above.</td></tr></tbody></table> <br><hr><br><h5><font color="#0000ff">Example macro 3:</font>
</h5>
<pre><font color="#000000"><font color="#ff0000"># EXAMPLE Experiment Minimization</font>
<font color="#ff0000"># Requires YASARA Dynamics</font>
Clear
LoadPDB 1crn
Clean
ForceField NOVA
Cutoff 10.48
Boundary Wall
<font color="#ff0000"># Prepare energy minimization experiment</font>
Experiment Minimization
<font color="#ff0000">  # Stop minimization as soon as the improvement per atom after 200 simulated annealing</font>
<font color="#ff0000">  # steps drops below 0.01 kJ/mol or kcal/mol (depending on current EnergyUnit).</font>
<font color="#ff0000">  # To skip simulated annealing completely, use 'Convergence immediate'.</font>
  Convergence 0.01
<font color="#ff0000"># Start experiment</font>
Experiment On
<font color="#ff0000"># Wait till end of experiment</font>
Wait ExpEnd
</font></pre>
  <table align="center"><tbody><tr><td width="400"><font color="#0000ff"><img src="Experiment%20-%20Choose%20and%20control%20experiments_files/ExperimentMinimization.jpg" align="middle" border="8"></font></td></tr><tr><td width="400">Figure: Result of the example macro
3 above.</td></tr></tbody></table> <br><hr><br><h5><font color="#0000ff">Example macro 4:</font>
</h5>
<pre><font color="#000000"><font color="#ff0000"># EXAMPLE Experiment Docking</font>
<font color="#ff0000"># Requires YASARA Structure</font>
Clear
<font color="#ff0000"># Load and clean the complex</font>
LoadPDB 1hvr,Download=yes
Stick
Style Ribbon
NiceOri
DelRes HYD
Clean
<font color="#ff0000"># Split receptor and ligand</font>
Split
<font color="#ff0000"># Join the protein chains in a single object</font>
JoinObj Protein,1
RenumberObj all
<font color="#ff0000"># Place the cell around the binding pocket (i.e. the ligand)</font>
RemoveObj 1
Cell Auto,Extension=8
AddObj 1
<font color="#ff0000"># Set the force field used to calculate ligand charges</font>
ForceField AMBER99
<font color="#ff0000"># Perform the docking</font>
Experiment Docking
<font color="#ff0000">  # Use AutoDockLS, a fast 'L'ocal 'S'earch to tune an existing complex,</font>
<font color="#ff0000">  # normally you would use AutoDockLGA, the 'L'amarckian 'G'enetic 'A'lgorithm.</font>
  Method AutoDockLS
<font color="#ff0000">  # Select receptor object</font>
  ReceptorObj 1
<font color="#ff0000">  # Select ligand object</font>
  LigandObj 2
<font color="#ff0000">  # Number of docking runs (normally 50-100)</font>
  Runs 1
<font color="#ff0000">  # Maximum RMSD allowed for ligands in the same cluster</font>
  ClusterRMSD 5.0
<font color="#ff0000">  # Name to save the various complexes, one per cluster, the number</font>
<font color="#ff0000">  # at the end of the filename will be incremented</font>
  ResultFile 1hvr_complex001
<font color="#ff0000">  # Additional docking parameters specified with 'DockPar' are simply added to</font>
<font color="#ff0000">  # the AutoDock input file:</font>
<font color="#ff0000">  # Uncomment below to set the number of energy evaluations:</font>
<font color="#ff0000">  # DockPar ga_num_evals 25000000</font>
<font color="#ff0000">  # Uncomment below to set the number of generations:</font>
<font color="#ff0000">  # DockPar ga_num_generations 27000</font>
<font color="#ff0000">  # Uncomment below to set the population size</font>
<font color="#ff0000">  # DockPar ga_pop_size 150</font>
<font color="#ff0000">  # Uncomment below to keep temporary files in the current working directory:</font>
<font color="#ff0000">  # TmpFileID YourChoice</font>
Experiment On
Wait ExpEnd
<font color="#ff0000"># Show close-up</font>
Pos X=0,Y=4,Z=46
Ori Alpha=-23,Beta=21,Gamma=-6
</font></pre>
  <table align="center"><tbody><tr><td width="400"><font color="#0000ff"><img src="Experiment%20-%20Choose%20and%20control%20experiments_files/ExperimentDocking.jpg" align="middle" border="8"></font></td></tr><tr><td width="400">Figure: Result of the example macro
4 above.</td></tr></tbody></table> <br><hr><br><h5><font color="#0000ff">Example macro 5:</font>
</h5>
<pre><font color="#000000"><font color="#ff0000"># EXAMPLE Experiment HomologyModeling</font>
<font color="#ff0000"># Requires YASARA Structure</font>
Clear
<font color="#ff0000"># Build homology model</font>
Experiment HomologyModeling
<font color="#ff0000">  # Specify the target sequence</font>
  SequenceFile target.fasta
<font color="#ff0000">  # Maximum oligomerization state</font>
  OligoState 2
<font color="#ff0000">  # Number of PSI-BLAST iterations</font>
  PsiBLASTs 6
<font color="#ff0000">  # The maximum PSI-BLAST Evalue (minimum can be set to avoid easy models)</font>
  EValue Max=0.5,Min=0
<font color="#ff0000">  # Maximum number of templates to consider, and how many with the same sequence</font>
  Templates 3,SameSeq=1
<font color="#ff0000">  # Custom templates that are not in the PDB must be loaded before and selected here:</font>
<font color="#ff0000">  #TemplateObj all</font>
<font color="#ff0000">  # Maximum number of ambiguous alignments to consider per template</font>
  Alignments 1
<font color="#ff0000">  # Alignments could also be provided:</font>
<font color="#ff0000">  #AlignFile alignments.fasta</font>
<font color="#ff0000">  # Number of samples to try per loop (see OptimizeLoop command)</font>
  LoopSamples 50
<font color="#ff0000">  # Maximum number of unaligned loop residues to add to N- and C-termini</font>
  TermExtension 10
<font color="#ff0000">  # Maximum and minimum number of residues (to avoid very large and small models)</font>
<font color="#ff0000">  #Residues Max=1500,Min=30</font>
<font color="#ff0000">  # Speed of animations</font>
  Animation Normal
<font color="#ff0000">  # The common start for result filenames</font>
  ResultFile model
Experiment On
</font></pre>
  <table align="center"><tbody><tr><td width="400"><font color="#0000ff"><img src="Experiment%20-%20Choose%20and%20control%20experiments_files/ExperimentHomologyModeling.jpg" align="middle" border="8"></font></td></tr><tr><td width="400">Figure: Result of the example macro
5 above.</td></tr></tbody></table> <br><hr><br><h5><font color="#0000ff">Example macro 6:</font>
</h5>
<pre><font color="#000000"><font color="#ff0000"># EXAMPLE Experiment NMRFolding</font>
<font color="#ff0000"># Requires YASARA Structure and the NMR Module</font>
Clear
<font color="#ff0000"># Build a linear peptide chain from the file sequence.fasta</font>
BuildMol sequence.fasta
Clean
<font color="#ff0000"># We need a forcefield with knowledge based potentials for sampling</font>
ForceField YASARA
SimSteps 500
<font color="#ff0000"># Fold the structures from the stretched out conformation</font>
Experiment NMRFolding
<font color="#ff0000">  # The object to fold</font>
  StartObj 1
<font color="#ff0000">  # The file(s) with NMR restraints (default extension is .tbl)</font>
  RestrainFile restraints.tbl
<font color="#ff0000">  # The number of structures to generate</font>
  Structures 1
<font color="#ff0000">  # The filename for the structures (digits will be incremented)</font>
  StructureFile ensemble001
<font color="#ff0000"># Generate the structure</font>
Experiment On
Wait 59
Style Tube
</font></pre>
  <table align="center"><tbody><tr><td width="400"><font color="#0000ff"><img src="Experiment%20-%20Choose%20and%20control%20experiments_files/ExperimentNMRFolding.jpg" align="middle" border="8"></font></td></tr><tr><td width="400">Figure: Result of the example macro
6 above.</td></tr></tbody></table> <br><hr><br><br>  </td></tr></tbody></table> 
</body></html>
