<html><head>
<meta http-equiv="content-type" content="text/html; charset=UTF-8">
</head>
<p><A HREF="index.html">Back</A></p>
<h1>References:</h1>
<P>
<A target ="_blank" HREF="http://dx.doi.org/10.1016/j.str.2011.08.006">Adams, P. D., Afonine, P. V., Bunk&oacute;czi, G., Chen, V. B., Davis, I.W., Echols, N., Headd, J. J., Hung, L.W., Kapral, G. J.,Grosse-Kunstleve, R. W., McCoy, A. J., Moriarty, N. W., Oeffner, R.,Read, R. J., Richardson, D. C., Richardson, J. S., Terwilliger, T. C.&amp; Zwart, P. H. (2010). PHENIX: a comprehensive Python-based system for macromolecular structure solution. <I>ActaCryst</I>. <B>D66</B>,213-221.</A><br><br>

<A target ="_blank" HREF="http://www.csb.yale.edu/userguides/datamanip/xplor/xplorman/htmlman.html">Br&uuml;nger, A. T. (1992). X-PLOR v.3.1. A system for X-ray Crystallography and NMR, Yale University, Connecticut, USA.</A><br><br>

<A target ="_blank" HREF="http://dx.doi.org/10.1107/S0907444998003254">Br&uuml;nger, A. T., Adams, P. D., Clore, G. M., DeLano, W. L., Gros, P.,Grosse-Kunstleve, R. W., Jiang, J.S., Kuszewski, J., Nilges, M.,Pannu, N.S., Read, R. J., Rice, L. M., Simonson, T. &amp;  Warren, G.L. (1998). Crystallography & NMR System: A New Software Suite for Macromolecular Structure Determination. <I>Acta Cryst.</I><B>D54</B>,905&ndash;921.</A><br><br>

<A target ="_blank" HREF="http://dx.doi.org/10.1107/S0907444904027325">Butterfoss, G. L., Richardson, J. S. &amp; Hermans, J. (2004). Protein imperfections: separating intrinsic from extrinsic variation of torsion angles. <I>Acta Cryst.</I><B>D61</B>, 88-98.</A><br><br>

<A target ="_blank" HREF="http://dx.doi.org/10.1016/S0079-6107(01)00005-0">Chakrabarti, P. &amp; Pal, D. (2001). The interrelationships of side-chain and main-chain conformations in proteins. <I>Prog. Biophys. Mol. Biol.</I> <B>76</B>,1-102. </A><br><br>

<A target ="_blank" HREF="http://dx.doi.org/10.1021/ja017712p">Clore, G. M., &amp; Kuszewski, J. (2002). &chi;1 Rotamer Populations and Angles of Mobile Surface Side Chains Are Accurately Predicted by a Torsion Angle Database Potential of Mean Force. <I>J. Am. Chem. Soc.</I> <B>124</B>,2866-2867.</A><br><br>

<A target ="_blank" HREF="http://dx.doi.org/10.1021/ja00124a002">Cornell, W. D., Cieplak, P., Bayly, C. I., Gould, I. R., Merz Jr., K. M.,Ferguson, D. M., Spellmeyer, D. C., Fox, T., Cardwell, J. W. &amp;Kollman, P. A. (1995). A second-generation force field for the simulation of proteins, nucleic acids, and organic molecules. <I>J.Am. Chem. Soc.</I>,<B>117</B>,5179-5197.</A><br><br>

<A target ="_blank" HREF="http://dx.doi.org/10.1093/nar/gkm216">Davis, I. W., Leaver-Fay, A., Chen, V. B., Block, J. N., Kapral, G. J., Wang, X., Murray, L. W., Arendall III, W. B., Snoeyink, J., Richardson, J. S. & Richardson, D. C. (2007). Molprobity: all-atom contacts and structure validation for proteins and nucleic acids. <I>Nucleic Acids Res.</I>,<B>35</B>,W375-W383.</A><br><br>

<A target ="_blank" HREF="http://dx.doi.org/10.1002/pro.5560060807">Dunbrack Jr., R. L. &amp; Cohen, F. E. (1997). Bayesian statistical analysis of protein side-chain rotamer preferences. <I>Prot. Sci</I>. <B>6</B>,1661-1681.</A><br><br>

<A target ="_blank" HREF="http://dx.doi.org/10.1006/jmbi.1993.1170">Dunbrack, R. L. &amp; Karplus, M. (1993). A backbone dependent rotamer library for proteins: application to sidechain prediction. <I>J.Mol. Biol.</I>,<B>230</B>,543-574.</A><br><br>

<A target ="_blank" HREF="http://dx.doi.org/10.1107/S0907444904019158">Emsley, P. &amp; Cowtan, K. (2004). Coot: model-building tools for molecular graphics. <I>Acta Cryst.</I> <B>D60</B>, 2126-2132.</A><br><br>

<A target ="_blank" HREF="http://dx.doi.org/10.1107/S0108767391001071">Engh, R. A. &amp; Huber, R. (1991). Accurate bond and angle parameters for X-ray protein structure refinement. <I>Acta Cryst.</I> <B>A47</B>, 392-400.</A><br><br>

<A target ="_blank" HREF="http://dx.doi.org/10.1093/nar/gkq453">Hekkelman, M. L., te Beek, T. A. H., Pettifer, S. R., Thorne, D., Attwood, T. K.&amp; Vriend, G. (2010). WIWS: a protein structure bioinformatics Web service collection. <I>Nucleic Acids Res.</I> <B>38</B>,W719-W723.</A><br><br>

<A target ="_blank" HREF="http://dx.doi.org/10.1021/ct700301q">Hess, B., Kutzner, C., van der Spoel, D. &amp; Lindahl, E. (2008). GROMACS 4:â€‰ Algorithms for Highly Efficient, Load-Balanced, and Scalable Molecular Simulation. <I>J.Chem. Theory Comput.</I>,<B>4</B>,435-447.</A><br><br>

<A target ="_blank" HREF="http://dx.doi.org/10.1093/bioinformatics/12.6.525">Hooft, R. W. W., Sander, C. &amp; Vriend, G. (1996a). The PDBFINDER database: a summary of PDB, DSSP and HSSP information with added value. <I>CABIOS</I> <B>12</B>,525-259.</A><br><br>

<A target ="_blank" HREF="http://dx.doi.org/10.1107/S0021889896008631">Hooft, R. W. W., Sander, C. &amp; Vriend, G. (1996b). Verification of Protein Structures: Side-Chain Planarity. <I>J. Appl. Cryst.</I><B>29</B>, 714-716.</A><br><br>

<A target ="_blank" HREF="http://dx.doi.org/10.1002/(SICI)1097-0134(199612)26:4<363::AID-PROT1>3.0.CO;2-D">Hooft, R. W. W., Sander, C. &amp; Vriend, G. (1996c). Positioning hydrogen atoms by optimizing hydrogen-bond networks in protein structures. <I>Proteins</I>,<B>26</B>,363-376.</A><br><br>

<A target ="_blank" HREF="http://dx.doi.org/10.1093/bioinformatics/13.4.425">Hooft, R. W. W., Sander, C. &amp; Vriend, G. (1997). Objectively judging the quality of a protein structure from a Ramachandran plot. <I>CABIOS</I> <B>13</B>,425-430.</A><br><br>

<A target ="_blank" HREF="http://dx.doi.org/10.1038/381272a0">Hooft, R. W. W., Vriend, G., Sander, C., &amp; Abola, E. E., (1996d). Errors in protein structures. <I>Nature</I><B>381</B>, 272.</A><br><br>

<A target ="_blank" HREF="http://dx.doi.org/10.1107/S090744490700978X">Jaskolski, M., Gilski, M., Dauter, Z. &amp; Wlodawer, A. (2007). Stereochemical restraints revisited: how accurate are refinement targets and how much should protein structures be allowed to deviate from them? <I>Acta Cryst.</I><B>D63</B>, 611-620.</A><br><br>

<A target ="_blank" HREF="http://dx.doi.org/10.1021/jp909088e">Jiang, F., Han, W. &amp; Wu, Y. (2010). Influence of Side Chain Conformations on Local Conformational Features of Amino Acids and Implication for Force Field Development. <I>J. Phys. Chem.</I> <B>B114</B>,5840-5850.</A><br><br>

<A target ="_blank" HREF="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC1166864/pdf/emboj00167-0183.pdf">Jones, T. A., &amp; Thirup, S., (1986). Using known substructures in protein model building and crystallography. <I>EMBO J.</I> <B>5</B>(4), 819-822.</A><br><br>

<A target ="_blank" HREF="http://dx.doi.org/10.1093%2Fbioinformatics%2Fbtr590">Joosten, R. P., Joosten, K., Cohen, S. X., Vriend, G. &amp; Perrakis, A. (2011). Automatic rebuilding and optimization of cyrstallographic structures in the Protein Data Bank. <I>Bioinformatics </I><B>27</B>,3392-3398</A><br><br>

<A target ="_blank" HREF="http://dx.doi.org/10.1126/science.317.5835.195 ">Joosten, R. P. &amp; Vriend, G. (2007). PDB Improvement Starts with Data Deposition. <I>Science</I>,<B>317</B>,195-196.</A><br><br>

<A target ="_blank" HREF="http://dx.doi.org/10.1093/nar/gkq1105">Joosten, R. P., te Beek, T. A. H., Krieger, E., Hekkelman, M. L., Hooft, R. W.W., Schneider, R., Sander, C. &amp; Vriend, G. (2011). A series of PDB related databases for everyday needs. <I>NucleicAcids Res.</I> <B>39</B>, D411-D419.</A><br><br>

<A target ="_blank" HREF="http://dx.doi.org/10.1002/bip.360221211">Kabsch, W. &amp; Sander, C. (1983). Dictionary of protein secondary structure: Pattern recognition of hydrogen-bonded and geometrical features. <I>Biopolymers</I> <B>22</B>, 2577-2637.</A><br><br>

<A target ="_blank" HREF="http://dx.doi.org/10.1002/pro.5560050719">Karplus, P. A. (1996). Experimentally observed conformation-dependent geometry and hidden strain in proteins. <I>Protein Science</I> <B>5</B>, 1406-142</A><br><br>

<A target ="_blank" HREF="http://www.gimp.org/">Kimball, S., Mattis, P., Natterer, M. &amp; Neumann, S. (1997) GIMP: GNU ImageManipulation Program, Version 2.6.8.</A><br><br>

<A target ="_blank" HREF="http://dx.doi.org/10.1107/S0907444904013253">Kleywegt, G. J., Harris, M. R., Zou, J., Taylor, T. C., W&auml;hlby, A. &amp;Jones, T. A. (2004). The Uppsala Electron-Density Server. <I>ActaCryst.</I> <B>D60</B>,2240-2249.</A><br><br>

<A target ="_blank" HREF="http://dx.doi.org/10.1007/978-1-61779-465-0_25">Krieger, E., Dunbrack Jr., R. L., Hooft, R. W., Krieger, B. (2012). Assignment of protonation states in proteins and ligands: combining pKa prediction with hydrogen bonding network optimization. <I>Methods Mol. Biol.</I><B>819</B>,405-421.</A><br><br>

<A target ="_blank" HREF="http://dx.doi.org/10.1002/prot.10104">Krieger, E., Koraimann, G. &amp; Vriend, G. (2002). Increasing the precision of comparative models with YASARA NOVA-a self-parameterizing force field. <I>Proteins: Struc. Func.Bioinf</I>. <B>47</B>, 393-402</A><br><br>

<A target ="_blank" HREF="http://dx.doi.org/10.1107/S0021889892009944">Laskowski, R. A., MacArthur, M. W., Moss, D. S., &amp; Thornton, J. M., (1993). PROCHECK: a program to check the stereochemical quality of protein structures. <I>J. App. Cryst.</I> <B>26</B>, 283-291.</A><br><br>

<A target ="_blank" HREF="http://dx.doi.org/10.1006/jmbi.1993.1351">Laskowski, R. A., Moss, D. S. &amp; Thornton, J. M. (1993). Main-chain Bond Lengths and Bond Angles in Protein Structures. <I>J. Mol. Biol.</I><B>231</B>, 1049-1067.</A><br><br>

<A target ="_blank" HREF="http://dx.doi.org/10.1002/prot.22711">Lindorff-Larsen, K., Piana, S., Palmo, K., Maragakis, P., Klepeis, J. L., Dror, R. O.&amp; Shaw, D. E. (2010). Improved side-chain torsion potentials for the Amber ff99SB protein force field. <I>Proteins: Struc. Func. Bioinf.</I> <B>78</B>,1950-1958.</A><br><br>

<A target ="_blank" HREF="http://dx.doi.org/10.1002/prot.10286">Lovell, S. C., Davis,  I. W., Arendall,  W. B., de Bakker, P. I. W., Word, J.M., Prisant, M. G., Richardson, J. S. &amp; Richardson, D. C. (2003). Structure validation by C&alpha; geometry: &#981;,&psi; and C&beta; deviation. <I>Proteins: Struc. Func. Bioinf.</I> <B>50</B>, 437-450.</A><br><br>

<A target ="_blank" HREF="http://dx.doi.org/10.1002/1097-0134(20000815)40:3%3C389::AID-PROT50%3E3.0.CO;2-2">Lovell, S. C., Word, J. M., Richardson, J. S. &amp; Richardson, D. C. (2000). The penultimate rotamer library. <I>Proteins: Struc. Func. Bioinf</I>. <B>40</B>, 389-408.</A><br><br>

<A target ="_blank" HREF="http://dx.doi.org/10.1103/PhysRevE.86.021904">Lundgren, M. &amp; Niemi, A. J. (2012). Correlation between protein secondary structure, backbone bond angles, and side-chain orientations. <I>Phys. Rev.</I> <B>E86</B>,021904(10).</A><br><br>

<A target ="_blank" HREF="http://dx.doi.org/10.1034/j.1600-0854.2000.010309.x">Luan, P., Heine, A., Zeng, K., Moyer, B., Greasely, S. E., Kuhn, P., Balch, W. E. &amp; Wilson, I. A. (2000). A New Functional Domain of Guanine Nucleotide Dissociation Inhibitor (&alpha;-GDI) involved in Rab Recycling. <I>Traffic</I> <B>1</B>,270-281.</A><br><br>

<A target ="_blank" HREF="http://books.google.nl/books?id=tN7JAceyJdUC&amp;printsec=frontcover&amp;hl=nl&amp;source=gbs_ge_summary_r&amp;cad=0#v=onepage&amp;q&amp;f=false">Lutz, M. (2001) Programming python, 2nd edition. O&rsquo;Reilly, Sebastopol</A><br><br>

<A target ="_blank" HREF="http://dx.doi.org/10.1006/jmbi.1996.0705">MacArthur, M. W. &amp; Thornton, J. M. (1996). Deviations from Planarity of the Peptide Bond in Peptides and Proteins. <I>J. Mol. Biol.</I> <I><B>264</B></I>,1180-1195.</A><br><br>

<A target ="_blank" HREF="http://dx.doi.org/10.1107/S0907444999002231">MacArthur, M. W. &amp; Thornton, J. M. (1999). Protein side-chain conformation: a systematic variation of 1 mean values with resolution - a consequence of multiple rotameric states? <I>Acta Cryst</I>. <B>D55</B>,994-1004.</A><br><br>

<A target ="_blank" HREF="http://dx.doi.org/10.1021/jp973084f">MacKerell Jr., A.D., Bashford, D., Bellott, M., Dunbrack, R. L., Evanseck, J.D., Field, M. J., Fischer, S., Gao, J., Guo, H., Ha, S.,Joseph-McCarthy, D., Kuchnir, L., Kuczera, K., Lau, F. T. K., Mattos,C., Michnick, S., Ngo, T., Nguyen, D. T., Prodhom, B., Reiher, W. E.,Roux, B., Schlenkrich, M., Smith, J. C., Stote, R., Straub, J.,Watanabe, M., Wiorkiewicz-Kuczera, J., Yin, D. &amp; Karplus, M.(1998). All-atom Empirical Potential for Molecular Modeling and Dynamics Studies of Proteins. <I>J.Phys. Chem. B.</I>,<B>102</B>,3586-3616.</A><br><br>

<A target ="_blank" HREF="http://dx.doi.org/10.1107/S0907444996012255">Murshudov,G. N., Vagin, A. A. &amp; Dodson, E. J. (1997). Refinement of Macromolecular Structures by the Maximum-Likelihood method. <I>ActaCryst.</I> <B>D53</B>,240&ndash;255.</A><br><br>

<A target ="_blank" HREF="http://dx.doi.org/10.1016/0022-2836(87)90358-5">Ponder, J. W. &amp; Richards, F. M. (1987). Tertiary templates for proteins. <I>J. Mol. Biol</I>. <B>193</B>,775-791.</A><br><br>

<A target ="_blank" HREF="http://www.R-project.org/">R Development Core Team (2012), R: A language and environment forstatistical computing, R Foundation for Statistical Computing,Vienna, Austria. SIBN 3-900051-07-0, URL http://www.R-project.org/</A><br><br>

<A target ="_blank" HREF="http://dx.doi.org/10.1016/j.str.2011.08.006">Read, R. J., Adams, P. D., Arendall, W. B., Br&uuml;nger, A. T., Emsley,P., Joosten, R. P., Kleywegt, G. J., Krissinel, E. B., L&uuml;tteke,T., Otwinowski, Z., Perrakis, A., Richardson, J. S., Sheffler, W. H.,Smith, J. L., Tickle, I. J., Vriend, G. &amp; Zwart, P. H. (2011). A New Generation of Crystallographic Validation Tools for the Protein Data Bank. <I>Structure</I> <B>19</B>, 1395-1412.</A><br><br>

<A target ="_blank" HREF="http://dx.doi.org/10.1021/bi011781z">Reiling, K. K., Endres, N. F., Dauber, D. S., Craik, C. S. &amp; Stroud, R. M.(2002). Anisotropic Dynamics of the JE-2147&minus;HIV Protease Complex:&nbsp; Drug Resistance and Thermodynamic Binding Mode Examined in a 1.09 &Aring; Structure. <I>Biochemistry</I> <B>41</B>, 4582-4594.</A><br><br>

<A target ="_blank" HREF="http://dx.doi.org/10.1002/prot.340190403">Rice, L. M. &amp; Br&uuml;nger, A. T. (1994). Torsion angle dynamics: Reduced variable conformational sampling enhances crystallographic structure refinement. <I>Proteins: Struc. Funct.Bioinf.</I> <B>19</B>, 277-290.</A><br><br>

<A target ="_blank" HREF="http://dx.doi.org/10.1016/0166-3542(94)00066-H">Rosenwirth, B., Oren, D.A., Arnold, E., Kis, Z.L. &amp; Eggers, H.J. (1995). SDZ 35-682, a new picornavirus capsid-binding agent with potent antiviral activity. <I>Antiviral Res.</I> <B>26</B>, 65-82.</A><br><br>

<A target ="_blank" HREF="http://dx.doi.org/10.1107/S1744309110052607">Schmidt, A., Teeter, M., Weckert, E. &amp; Lamzin, V. S. (2011). Crystal structure of small protein crambin at 0.48&nbsp;&Aring; resolution.<I> ActaCryst.</I> <B>F67</B>, 424-429.</A><br><br>

<A target ="_blank" HREF="http://www.pymol.org/">Schr&ouml;dinger, L. L. C. (2010) The PyMOL Molecular Graphics System, Version 1.2.0.1 </A><br><br>

<A target ="_blank" HREF="http://dx.doi.org/10.1107/S0108767307043930">Sheldrick, G. M. (2008). A short history of&nbsp;SHELX. <I>ActaCryst.</I> <B>A64</B>,112&ndash;122.</A><br><br>

<A target ="_blank" HREF="http://dx.doi.org/10.1093/nar/gkp783">Sychrovsky, V., Foldynova-Trantirkova, S., Spackova, N., Robeyns, K., vanMeervelt, L., Blankenfeldt, W., Vokacova, Z., Sponer, J. &amp;Trantirek, L. (2009). Revisiting the planarity of nucleic acid bases: Pyramidilization at glycosidic nitrogen in purine bases <I>Nucleic Acids Res.</I> <B>37</B>, 7321-7331.</A><br><br>

<A target ="_blank" HREF="http://dx.doi.org/10.1107/S0907444910040928">Touw, W. G. &amp; Vriend, G. (2010). On the complexity of Engh and Huber refinement restraints: the angle as example. <I>Acta Cryst</I>. <B>D66</B>,1341-1350</A><br><br>

<A target ="_blank" HREF="http://dx.doi.org/10.1002/0470080124.ch1">Tsai, C. S. (2007) Biomacromolecules: Introduction to Structure, Functionand Informatics, Hoboken, John Wiley &amp; Sons, chapter 1, pp 4, NewJersey.</A><br><br>

<A target ="_blank" HREF="http://dx.doi.org/10.1016/0263-7855(90)80070-V">Vriend, G. (1990). WHAT IF: A molecular modeling and drug design program. <I>J. Mol. Graph.</I> <B>8</B>, 52-56.</A><br><br>

<A target ="_blank" HREF="http://dx.doi.org/10.1038/nprot.2008.184">Wang, Q., Canutescu, A. A., &amp; Dunbrack Jr., R.L. (2008). SCWRL and MolIDE: computer programs for side-chain conformation prediction and homology modeling. <I>Nat. Protoc.</I><B>3</B>, 1832-1847.</A><br><br>

<A target ="_blank" HREF="http://www.jstatsoft.org/v40/i01">Wickham, H. (2011), <I>Journal of Statistical Software</I>, The Split-Apply-CombineStrategy for Data Analysis, 40, 1-29,http://www.jstatsoft.org-v40-i01.</A><br><br>
</P>
<h2>Further reading:</h2>
<P><A HREF="http://dx.doi.org/10.1002/1097-0134(20000815)40:3%3C389::AID-PROT50%3E3.0.CO;2-2">
Lovell, S. C., Word, J. M., Richardson, J.S., Richardson, D.C. (2000). The penultimate rotamer library. <I> Proteins.</I>
<B>40</B>,
389&ndash;408</A></P>
<P><A HREF="http://dx.doi.org/10.1006/jmbi.1998.1852">
Markley, J. L., Bax, A., Arata, Y., Hilbers, C. W., Kaptein, R., Sykes, B. D., Wright, P. E., W&uuml;thrich, K. (1998). 
Recommendations for the presentation of NMR structures of proteins and nucleic acids. <I> J. Mol. Biol.</I>
<B>280</B>,933&ndash;952.
</BODY>
</HTML>
